cardano-node/src/Cardano/Node/Types.hs

```diff
@@ -545,7 +545,7 @@ parseNodeConfigurationFP :: ConfigYamlFilePath -> IO NodeConfiguration
 parseNodeConfigurationFP (ConfigYamlFilePath fp) = do
     nc <- decodeFileThrow fp
     -- Make all the files be relative to the location of the config file.
-    pure $ adjustFilePaths (takeDirectory fp </>) nc
+    pure $ adjustFilePaths (\p -> takeDirectory fp <> "/" <> p) nc
```

Comment:
Instead of constructing this kind of string on Windows (not the backslash):

```
source/configuration/defaults/simpleview\genesis/genesis.json
```